src/zenml/post_execution/pipeline_run.py

```diff
@@ -147,12 +147,14 @@ def extra(self) -> Dict[str, Any]:
         return cast(Dict[str, Any], extra)
 
     @property
-    def enable_cache(self) -> bool:
+    def enable_cache(self) -> Optional[bool]:
         """Returns whether caching is enabled for this pipeline run.
 
         Returns:
             True if caching is enabled for this pipeline run.
         """
+        if "enable_cache" not in self.pipeline_configuration:
```

Comment:
Worth refactoring this out to a constant somewhere? We've been burned by these strings in the past.

Reply:
We even have this as a constant in `base_pipeline` already, I imported and used that one instead of the strings 👍